frontend/src/store/profile/action.ts

```diff
@@ -145,7 +145,7 @@ export const updateCurrentUser = (newUser: User) => async (dispatch: Dispatch<Ac
   }
 
   // 2. Update user profile
-  await fetch(DOMAIN_URL.SOCIAL + ENDPOINTS.EDIT_PROFILE, {
+  await fetch(DOMAIN_URL.SOCIAL + ENDPOINTS.EDIT_PROFILE + `?token=${localStorage.getItem('token')}`, {
```

Comment:
for thought https://washpost.engineering/string-manipulation-of-urls-is-an-anti-pattern-e2b341129025

Reply:
fixed in 0be3c601d7535f6efb7bc26931ce709988a83151